core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -1124,7 +1220,7 @@ public static MediaType parse(String input) {
                 tokenizer.consumeTokenIfPresent(LINEAR_WHITE_SPACE);
                 String attribute = tokenizer.consumeToken(TOKEN_MATCHER);
                 tokenizer.consumeCharacter('=');
-                final String value;
+                String value;
```

Comment:
Q) This is more of a question rather than a comment. Is there a reason why the `final` modifier is removed here (not only here but for this file overall). Is it because this class was mostly forked (and will make future forking easier)?

Reply:
Yes. The upstream removed `final` from this line, so I made the same change so we can keep it as close as possible to the upstream.